pubnub-util/src/uritemplate.rs

```diff
@@ -48,21 +48,21 @@ impl UriTemplate {
     }
 
     /// Bind a variable to a list value.
-    pub fn set_list<T: Into<String>>(
+    pub fn set_list<T: ToString>(
         &mut self,
         varname: impl AsRef<str>,
         var: impl IntoIterator<Item = T>,
     ) -> &mut Self {
         self.0.set(
             varname.as_ref(),
-            TemplateVar::List(var.into_iter().map(Into::into).collect()),
+            TemplateVar::List(var.into_iter().map(|val| val.to_string()).collect()),
```

Comment:
`ToString::to_string` ought to work here.

```suggestion
            TemplateVar::List(var.into_iter().map(ToString::to_string).collect()),
```

Reply:
There's a lifetime mismatch there, so it doesn't work. I tried doing it like that too at first.

Reply:
Oof! That's too bad. No big deal.